episodes/03-hello-open-source.md

```diff
@@ -6,22 +6,216 @@ exercises: 20
+academic citations. Most people have had the experience of following up a reference to a website and either getting a
+404 error or seeing something completely different. Although using versions on your site doesn't guarantee this won't
+happen, it does make it easier to manage old versions of your site.
+
+GitHub Pages also mean that you can collaborate on a website with a lot of people without everyone having to
+communicate endlessly back and forwards about what changes need to be made, or have been made already. You can create
+'issues' (things that need discussing or fixing), list things to do in the future, and allow other people visiting your
+website to quickly suggest, and help implement changes through pull requests.
+
+### Creating a new repository
+
+To create a new repo login to your GitHub account. At the top, right corner click on the `+` sign and choose `New repository`
+
+![](fig\gh_new-repo.png){alt="GitHub create a new repo button"}
+
+
+Give your repository a meaningful name that suits the contents it shall hold. And we encourage adding something to the optional description section. 
+Toggle the `Add a README file` button and click on the green `Create Repository` button.
+
+![](fig\gh-new-repo-name.png){alt="GitHub create a new repo page"}
+
+
+Here is a preview of how the new repo would look like:
+
+![](fig\gh_new-repo-page.png){alt="GitHub new repo view"}
+
+
+### Enable GitHub Pages
+
+GitHub Pages is turned off by default for all new repositories, and can be turned on in the settings menu for any repository.
+
+Let's set up a new site by enabling GitHub Pages for our project.
+
+Go to the Pages section of your repository's Settings:
+
+![](fig/github-repo-settings-pages.png){alt="GitHub repository's Pages settings"}
+
+#### Source branch (required)
+
+Pages needs to know the branch in your repository from which you want to serve your site. This can be any branch, including `main`.
+
+Select then save the source branch:
+
+![](fig/github-repo-settings-pages-branch.png){alt="GitHub Pages source branch menu"}
+
+![](fig/github-repo-settings-pages-save.png){alt="GitHub Pages source branch save"}
+
+#### Theme (optional)
+
+GitHub Pages provides different themes to visually style and organize your site's content. Choosing a theme is optional, and themes can be interchanged quickly.
+
+![](fig/github-repo-settings-pages-theme.png){alt="GitHub Pages choose theme"}
+
+See the [GitHub Pages documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/adding-a-theme-to-your-github-pages-site-with-the-theme-chooser) for further information on using themes.
+
+### View your site
+
+If we now visit `https://some-humanist.github.io/my-website/`,
+we should see the contents of the index.md file that created earlier.
+Usually it's available instantly, but it can take a few seconds and in the worst case a few minutes if GitHub are very busy.
+
+And here's a preview of the new website
+
```

Comment:
break this down, my usernamd is x, my repo is named y, then my site is ...

hav ea challenge block to view their own website